### PR TITLE
Feature/footer

### PR DESCRIPTION
**Pull request by Drelofs**
Merging footer with grid

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ module.exports = function(grunt) {
 		watch: {
 			sass: {
 				// We watch and compile sass files as normal but don't live reload here
-				files: ['src/assets/**/*.scss'],
+				files: ['src/assets/style.scss'],
```
Comment (matthijs186) [inline Gruntfile.js]: [Change](https://github.com/peterxqe/teamfairtrade/pull/3/commits/b595bf35a9d57691b7cb13f26252d74c0fc178db) is OK.

**Review by RishiTheGoat — APPROVED** [review]
Looks good Ali! 🗡

**Review by peterxqe — APPROVED** [review]
ait 10/10

File: src/assets/partials/_footer.scss [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+		margin: 6% 5px 0 10px;
+	}
+
+	.page-bottom-list {
+		padding: 0;
+	  	margin: 0;
+	}
+}
+
+@media screen and (max-width: 640px) {
```
Comment (matthijs186) [inline src/assets/partials/_footer.scss]: What is this `640px`?

**Review by matthijs186 — CHANGES_REQUESTED** [review]
Please make above changes.

**Review by matthijs186 — APPROVED** [review]
Review complete, good to go!

**Review by peterxqe — APPROVED** [review]
is goed!

**Review by ferrybig — APPROVED** [review]

**Pull request merged**